wgpu-types/src/lib.rs

```diff
@@ -5626,6 +5626,8 @@ pub enum SurfaceStatus {
     Outdated,
     /// The surface under the swap chain is lost.
     Lost,
+    /// The surface status is not known.
+    Unknown,
```

Comment:
I'm wondering if  `Unknown` is really the right term here. We know that `get_current_texture` failed for a known reason when we get this. But then again we don't know what state our surface is in at this point (the error may have caused mayhem such that reconfigure is in order or the next `get_current_texture` call might just succeed).

So maybe it's enough to just clarify this here a little bit:

```suggestion
    /// The surface status is not known since `get_current_texture` previously failed.
    Unknown,
```

Reply:
I like the idea of clarifying via the comment. The reason I chose `Unknown` as the name here is exactly what you mentioned: we don't really know the status of the surface when we get this error.

For example, if we get a `DeviceLost` error we can't reliably report anything about the status of the Surface (since we got an error higher in the stack)